client/me/account-close/confirm-dialog.jsx

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import FormLabel from 'components/forms/form-label';
+import Button from 'components/button';
+
+const AccountCloseConfirmDialog = ( {
+	isVisible,
+	username,
+	onConfirm,
+	translate,
+	closeConfirmDialog,
+	deleteAccount,
+} ) => {
+	const deleteButtons = [
+		<Button onClick={ closeConfirmDialog }>{ translate( 'Cancel' ) }</Button>,
+		<Button primary scary onClick={ deleteAccount }>
+			{ translate( 'Close your account' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Close your account permanently' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).